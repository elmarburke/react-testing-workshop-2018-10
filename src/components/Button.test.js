import React from 'react';
import Button from './Button';
import { render } from 'react-testing-library';

describe('<Button />', () => {
  test('basic render', () => {
    // Arrange
    const { container, asFragment } = render(<Button>Send Data</Button>);
    // Act
    // Assert
    expect(container).toHaveTextContent('Send Data');
    // expect(container.firstChild).toMatchSnapshot()
  });
});
