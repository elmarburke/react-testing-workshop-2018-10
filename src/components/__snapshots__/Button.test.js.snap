// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> basic render 1`] = `
<button>
  Send Data!
</button>
`;
